proto/cosmos/staking/v1beta1/tx.proto

```diff
@@ -30,24 +30,51 @@ service Msg {
   // Undelegate defines a method for performing an undelegation from a
   // delegate and a validator.
   rpc Undelegate(MsgUndelegate) returns (MsgUndelegateResponse);
+
+  // UnbondValidator defines a method for performing the status transition for a validator
+  // from bonded to unbonded
+  rpc UnbondValidator(MsgUnbondValidator) returns (MsgUnbondValidatorResponse);
+
+  // CancelUnbondingDelegation defines a method for performing canceling the unbonding delegation
+  // and delegate back to previous validator.
+  //
+  // Since: cosmos-sdk 0.46
+  rpc CancelUnbondingDelegation(MsgCancelUnbondingDelegation) returns (MsgCancelUnbondingDelegationResponse);
```

Comment:
Is backporting `CancelUnbondingDelegation` rpc and associated messages to`cosmos-sdk v45` required to support LSM module operation?

Reply:
It is desirable in the LSM on SDK v45 as a safety feature. If a liquid staking provider is exploited and the exploiter initiates undelegation, having access to `CancelUnbondingDelegation` allows the liquid staking provider to cancel undelegation after software upgrade and thus avoid loss of user funds.

Reply:
added better docs [[here](https://github.com/cosmos/cosmos-sdk/pull/16747/commits/69e6bb7a41eae0f4eaced4a049e172c0bbfdf4d8)] (ty @riley-stride  for the summary!)

Reply:
Awesome, thanks for elaborating!

We will have to make sure the ICS provider cannot be negatively affected by the change.
I'll try to simulate the scenario

Reply:
ah great point. That'd be great if you could simulate it! 💪